src/platform_impl/macos/view.rs

```diff
@@ -1126,3 +1140,34 @@ fn replace_event(event: &NSEvent, option_as_alt: OptionAsAlt) -> Id<NSEvent> {
         event.copy()
     }
 }
+
+// Convert given NSRange to Rust String Range data used in this crate.
+// Steps:
+// 1. Use the given preedit text to initialize an NSString instance.
```

Comment:
As the logic of this code is obvious, I think you don't need to describe it in your documentation again.

Reply:
That's requested by @madsmtm .